src/main/java/net/pms/formats/Format.java

```diff
@@ -422,7 +422,7 @@ public static boolean isSupportedMimetype(String mimetype) {
 		if ("audio/L24".equalsIgnoreCase(mimetype)
 				|| "video/vnd.dlna.mpeg-tts".equalsIgnoreCase(mimetype)
 				|| "video/mpeg2".equalsIgnoreCase(mimetype)
-				|| "audio/vnd.dlna.adts".equals(mimetype)
+				|| "audio/vnd.dlna.adts".equalsIgnoreCase(mimetype)
```

Comment:
@Sami32 That will result in less robust code - anyone edition will have to know/remember this rule or create a bug. A much better solution (imo) is to use ```Enum``` for things like that, it's type-safe and evaluation is much quicker.